android/src/main/java/com/facebook/flipper/plugins/databases/DatabasesManager.java

```diff
@@ -0,0 +1,202 @@
+package com.facebook.flipper.plugins.databases;
+
+import android.util.SparseArray;
+import com.facebook.flipper.core.FlipperArray;
+import com.facebook.flipper.core.FlipperConnection;
+import com.facebook.flipper.core.FlipperObject;
+import com.facebook.flipper.core.FlipperReceiver;
+import com.facebook.flipper.core.FlipperResponder;
+import com.facebook.flipper.plugins.databases.DatabaseDriver.DatabaseExecuteSqlResponse;
+import com.facebook.flipper.plugins.databases.DatabaseDriver.DatabaseGetTableDataResponse;
+import com.facebook.flipper.plugins.databases.DatabaseDriver.DatabaseGetTableStructureResponse;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
+
+public class DatabasesManager {
+
+    private final static String DATABASE_LIST_COMMAND = "databaseList";
+    private final static String GET_TABLE_DATA_COMMAND = "getTableData";
+    private final static String GET_TABLE_STRUCTURE_COMMAND = "getTableStructure";
+    private final static String EXECUTE_COMMAND = "execute";
+
+    private final List<DatabaseDriver> databaseDriverList;
+
+    private final SparseArray<DatabaseDescriptorHolder> databaseDescriptorHolderSparseArray;
+    private final Set<DatabaseDescriptorHolder> databaseDescriptorHolderSet;
+
+    private FlipperConnection connection;
```

Comment:
Naming scheme here doesn't match that of other files. Should be `mConnection`, `mDatabaseDriverList`, etc.